www/html/browserunsup.ejs

```diff
@@ -1,3 +1,4 @@
 <% include partials/head %>
 
-<h1>Please use <a href='https://www.google.com/chrome/'>Google Chrome</a> or <a href='https://www.mozilla.org/en-US/firefox/new/'>Mozilla Firefox</a> in order to view this webpage</h1>
\ No newline at end of file
+<h1>Please use <a href='https://www.google.com/chrome/'>Google Chrome</a> or <a href='https://www.mozilla.org/en-US/firefox/new/'>Mozilla Firefox</a> in order to view this webpage</h1>
+<% include partials/footer %>
```

Comment:
Added footer

Reply:
I intentionally did not add the footer here.

Reply:
Actuslly nevermind.